patches/tModLoader/Terraria/Player.cs.patch

```diff
@@ -2905,15 +2905,16 @@
  			if (mount.Active) {
  				legFrameCounter = 0.0;
  				legFrame.Y = legFrame.Height * 6;
-@@ -25526,6 +_,8 @@
- 						}
- 					}
- 				}
-+				else
```

Comment:
Review from discord:
The hook should come at the end so that if someone wants to do a final tweak of piggyback  vanilla use style, they can.
Should wait to update until after Mirsario use item changes are done to ensure PR design relevancy.